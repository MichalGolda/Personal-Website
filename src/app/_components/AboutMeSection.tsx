"use client";

import { SectionHeading, Container } from "@/app/_components";
import { useInViewSection } from "../_context/inViewSectionContext";

export default function AboutMeSection() {
  const { ref } = useInViewSection("#about-me");

  return (
    <section ref={ref} id="about-me" className="mt-16 mb-64">
      <Container>
        <SectionHeading headline="O mnie" title="Kilka słów o tym kim jestem" />
        <p className="mt-16 text-body max-w-3xl text-sm md:text-base mx-auto text-center">
          Cześć, jestem Michał. Od kilku lat interesuje się programowaniem i
          tworzeniem stron internetowych. Swoją dotychczasową wiedzę poszerzałem
          głównie poprzez realizację projektów we własnym zakresie. Staram się
          być na bieżąco z aktualnymi trendami na rynku, zdobywać nową wiedzę,
          aby dostarczać rozwiązania spełniające aktualne standardy jakości.
        </p>
      </Container>
    </section>
  );
}
