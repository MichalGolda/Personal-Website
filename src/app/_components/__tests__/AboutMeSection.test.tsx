import "@/app/_mocks/intersectionObserver.mock";
import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import AboutMeSection from "../AboutMeSection";

describe("AboutMeSection component", () => {
  it("renders unchanged", () => {
    const { container } = render(<AboutMeSection />);

    expect(container).toMatchSnapshot();
  });
});
