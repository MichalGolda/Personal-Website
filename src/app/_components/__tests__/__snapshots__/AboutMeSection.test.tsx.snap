// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMeSection component renders unchanged 1`] = `
<div>
  <section
    class="mt-16 mb-64"
    id="about-me"
  >
    <div
      class="md:max-w-[1024px] lg:max-w-[1320px] w-full mx-auto md:px-16 px-4 "
    >
      <div
        class="text-center max-w-3xl mx-auto"
      >
        <span
          class="text-primary font-semibold text-md md:text-lg lg:text-xl uppercase"
        >
          O mnie
        </span>
        <h2
          class="text-2xl md:text-3xl lg:text-4xl text-secondary font-bold"
        >
          Kilka słów o tym kim jestem
        </h2>
      </div>
      <p
        class="mt-16 text-body max-w-3xl text-sm md:text-base mx-auto text-center"
      >
        Cześć, jestem Michał. Od kilku lat interesuje się programowaniem i tworzeniem stron internetowych. Swoją dotychczasową wiedzę poszerzałem głównie poprzez realizację projektów we własnym zakresie. Staram się być na bieżąco z aktualnymi trendami na rynku, zdobywać nową wiedzę, aby dostarczać rozwiązania spełniające aktualne standardy jakości.
      </p>
    </div>
  </section>
</div>
`;
